src/app/app-routing.module.ts

```diff
@@ -59,8 +59,8 @@ const routes: Routes = [
       { path: '', component: WirelessDetailComponent }
     ]
   },
-  { path: 'wireless/:name', component: WirelessDetailComponent },
-  { path: 'event-channel', component: EventChannelComponent }
+  { path: 'wireless/:name', component: WirelessDetailComponent }
+  // { path: 'event-channel', component: EventChannelComponent }
```

Comment:
Do you want to remove the commented line?

Reply:
Not particularly, normally, yes. However, i've essentially commented out this entire feature until `ngx-mqtt` can address the CVE. Until then, ripping it out.